UPGRADE-6.3.md

```diff
@@ -0,0 +1,7 @@
+UPGRADE FROM 6.2 to 6.3
+=======================
+
+Filesystem
+----
```

Comment:
Just a minor, but the length of the underline, should match the length of the headline

Reply:
Done (y)